Projects/App/Sources/UI/Organization/OrganizationList/ViewModel/OrganizatinoListViewModel.swift

```diff
@@ -77,18 +77,21 @@ extension OrganizationListViewModel: ErrorMapper {
             searchedOrgArray = []
             return
         }
-        if input.contains(" ") {
+        
+        let inputUppercase = input.uppercased()
+        
+        if inputUppercase.contains(" ") {
             let inputArray = input.components(separatedBy: " ")
 
             searchedOrgArray = orgArray.filter { organization in
                 var haveOrgName = false
-                for input in inputArray where organization.name.contains(input) {
+                for input in inputArray where organization.name.contains(input.uppercased()) {
```

Comment:
input을 모두 uppercase로 변환한 후에 포함되는지 판단하는 방식 말고 case에 상관없이 포함됐는지 판단하는 `localizedCaseInsensitiveContains`를 사용하는건 어떤가요?
이러면 inputUppercase 상수를 만들어줄 필요 없이 원래 코드에서 `.contains` 두 개만 바꾸면 해결됩니다!

```suggestion
                for input in inputArray where organization.name.localizedCaseInsensitiveContains(input.uppercased()) {
```

Reply:
오 이런 방법이
수정했습니다 546f61e60540743ac825ee51195e6fd1795ed274